web/ui-go/Makefile

```diff
@@ -0,0 +1,19 @@
+PROTO_DIR := ../../core/lib/src/main/java/dev/enola/core
+PROTO_FILE := enola_core.proto
+GO_OUT := generated
+
+.PHONY: all
+all: generate
+
+.PHONY: generate
+generate:
+	@echo "Generating Go code from protobuf..."
+	protoc --proto_path=$(PROTO_DIR) --go_out=$(GO_OUT) $(PROTO_DIR)/$(PROTO_FILE)
+	protoc --proto_path=$(PROTO_DIR) --go-grpc_out=$(GO_OUT) $(PROTO_DIR)/$(PROTO_FILE)
+	@echo "Go code generated successfully."
```

Comment:
As far as I understand, what @digitalentity contributed in #227 should mean we don't need this?

Reply:
So you should be able to just have a `deps` for `//core/lib:core_go_proto` on the `go_binary` in `web/ui-go/BUILD`